CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ## Unreleased
 
+### Changed
+
+- Timestamp argument accepts input in string type( format `YYYY-MM-DD HH:MM:SS` ), int type(epoch time) as well as a datetime instance for following:
+    - `sdk.auditlogs.get_page`
```

Comment:
Probably should specify which args this affect for each method

Reply:
Done.